psi4/src/psi4/libscf_solver/rhf.cc

```diff
@@ -48,13 +49,26 @@
 #include "psi4/libiwl/iwl.hpp"
 #include "psi4/libmints/factory.h"
 #include "psi4/libmints/matrix.h"
+#include "psi4/libmints/molecule.h"
+#include "psi4/libmints/pointgrp.h"
 #include "psi4/liboptions/liboptions.h"
 #include "psi4/libpsi4util/PsiOutStream.h"
 #include "psi4/libpsi4util/process.h"
 #include "psi4/libpsio/psio.h"
 #include "psi4/libqt/qt.h"
 #include "psi4/libtrans/integraltransform.h"
 
+#ifdef USING_OpenOrbitalOptimizer
+#ifdef USING_LAPACK_MKL
+#include <mkl.h>
+#define ARMA_USE_MKL
+#define ARMA_USE_MKL_TYPES
+#endif
+#define ARMA_DONT_USE_FORTRAN_HIDDEN_ARGS
+#define ARMA_DONT_USE_WRAPPER
```

Comment:
this is already done by the psi4/libmints/matrix.h include, no? no need to duplicate the logic here